ci-operator/config/openshift/cincinnati-operator/openshift-cincinnati-operator-master.yaml

```diff
@@ -23,6 +27,21 @@ images:
 - dockerfile_path: dev/Dockerfile
   from: base
   to: cincinnati-graph-data-container
+operator:
+  bundles:
+  - as: cincinnati-bundle-container
+    base_index: operator-index
+    context_dir: config
+    dockerfile_path: bundle.Dockerfile
```

Comment:
I didn't see this file under https://github.com/openshift/cincinnati-operator/tree/master/config